.eslintrc.js

```diff
@@ -13,5 +16,8 @@ module.exports = {
         "sourceType": "module"
     },
     "rules": {
-    }
+      "no-console":0,
```

Comment:
```suggestion
      "no-console": "off",
```
Just keeping things consistent!

Reply:
for the `"no-console":` when I write `"off"` it throws an error that `0 = "off"`.  It means we will have to stick with `"no-console": 0`